Sources/UIViewAPI/UIViewAPI.swift

```diff
@@ -20,4 +20,24 @@ public extension UIView {
     }
 }
 
+public extension UILabel {
+    var getIntrinsicLabelWidth: CGFloat {
+        get { self.intrinsicContentSize.width }
+    }
+    
+    var getLabelWidth: CGFloat {
+        get { self.frame.width }
+    }
+    
+    func numberOfLines() -> Int {
```

Comment:
Please add a documentation what it does and a example snippet :)

Reply:
Sure, added description in documentation